english/eating-walls/find-a-toilet/relieve/relieve-yourself.md

```diff
@@ -1,3 +1,9 @@
-Gross!
+Thank the gods! Your stomach stops grumbling. 
 
-You turn and run away in shame.
+However, you've reached a moral crossroads: 
+
+Should you clean up the horrible mess you've made or leave it for someone
+else to deal with?
+
+[Clean up?](clean-up.md)
+[Leave 'er there?](leave-poop.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.